libsel4/sel4_arch_include/aarch32/sel4/sel4_arch/constants.h

```diff
@@ -155,6 +155,34 @@ enum {
     seL4_TimeoutReply_Length,
     SEL4_FORCE_LONG_ENUM(seL4_TimeoutReply_Msg)
 } seL4_TimeoutReply_Msg;
+
```

Comment:
do you consider encoding this into the existing temporal fault rather than introducing a new fault type?

Reply:
Given they are represented much the same and are handled basically identically, that is an entirely good option. Would mean extending the existing temporal fault with the `reason`, but I think that makes a lot of sense.

Reply:
I highly recommend you do this. It will reduce the added code and total code size.